Add missing ClusterRoleBinding and leases permissions for running 2nd scheduler (#20691)
kubernetes/website: content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md

You should see a "Running" my-scheduler pod, in addition to the default kube-scheduler
pod in this list.

### Enable leader election

To run multiple-scheduler with leader election enabled, you must do the following:

First, update the following fields in your YAML file:

* `--leader-elect=true`
* `--lock-object-namespace=<lock-object-namespace>`
* `--lock-object-name=<lock-object-name>`

> Note that the objects will be created, but the namespace must already exists.
> You can use the `kube-system` namespace.

Reviewer: I would use a note shortcode:
```suggestion
{{< note >}}
The control plane creates the lock objects for you, but the namespace must already exist.
You can use the `kube-system` namespace.
{{< /note >}}
```

Author: Done

> If RBAC is enabled on your cluster, you must update the `system:kube-scheduler` cluster role. Add your scheduler name to the resourceNames of the rule applied for `endpoints` and `leases` resources, as in the following example:

Reviewer: How does this interact with EndpointSlice?

Author: kube-scheduler doesn't use them.
